src/site/content/es-es/vitals/index.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: page
+title: Web Vitals
+subhead: Métricas esenciales para un sitio saludable
+description: Métricas esenciales para un sitio saludable
+authors:
+  - philipwalton
+date: 30-04-2020
+updated: 21-07-2020
+masthead: web-vitals.svg
+tags:
+  - metrics
+  - performance
+  - web-vitals
+---
+
+Optimizar la calidad de la experiencia del usuario es un factor clave para el éxito a largo plazo de cualquier sitio web. Tanto si es propietario de un negocio, especialista de marketing o desarrollador, los Web Vitals pueden ayudarle a cuantificar la experiencia de su sitio e identificar oportunidades para mejorar.
+
+## Descripción general
+
+Web Vitals es una iniciativa de Google para proporcionar una guía unificada de indicadores de calidad que son esenciales para brindar una excelente experiencia de usuario en la web.
+
+Google ha proporcionado una serie de herramientas a lo largo de los años para medir y generar informes de rendimiento. Algunos desarrolladores son expertos en el uso de estas herramientas, mientras que a otros les resulta difícil mantenerse al día con la enorme cantidad de herramientas y métricas disponibles.
+
+Los propietarios de sitios no deberían tener que ser gurús del rendimiento para comprender la calidad de la experiencia que brindan a sus usuarios. La iniciativa Web Vitals tiene como objetivo simplificar el panorama y ayudar a los sitios a centrarse en las métricas que más importan, los llamados **Core Web Vitals**.
+
+## Core Web Vitals
+
+Los Core Web Vitals son el subconjunto de Web Vitals que se aplica a todas las páginas web, todos los propietarios de sitios deben medirlos y aparecerán en todas las herramientas de Google. Cada uno de los Core Web Vitals representa una faceta distinta de la experiencia del usuario, se puede medir [sobre el terreno](/user-centric-performance-metrics/#how-metrics-are-measured) y refleja la experiencia en el mundo real de un resultado crítico [centrado en el usuario](/user-centric-performance-metrics/#how-metrics-are-measured).
+
+Las métricas que componen los Core Web Vitals [evolucionarán](#evolving-web-vitals) con el tiempo. El conjunto actual para 2020 se centra en tres aspectos de la experiencia del usuario: *carga* , *interactividad* y *estabilidad visual*, e incluye las siguientes métricas (y sus respectivos umbrales):
+
+<div class="w-stack w-stack--center w-stack--md">
+<img src="lcp_ux.svg" width="400px" height="350px" alt="Las recomendaciones de umbral de pintura con contenido más grandes"><img src="fid_ux.svg" width="400px" height="350px" alt="Recomendaciones de umbral de retardo de primera entrada"><img src="cls_ux.svg" width="400px" height="350px" alt="Recomendaciones de umbral de cambio de diseño acumulativo">
+</div>
+
```

Comment:
Bellow is the first time that the metrics are mentioned and we should show what the words that compose the metric names mean:

Largest Contentful Paint : Despiliegue del contenido más grande
First Input Delay: Demora de la primera entrada
Cumulative Layout Shift: Cambio Acumulativo del diseño